frontend/src/scenes/constructor/containers/index.tsx

```diff
@@ -303,33 +303,43 @@ const Constructor: React.FC<IProps> = ({ id }) => {
             <Form.Label className={style.row} />
             <Form.Group controlId="Resource" className={style.resource}>
               <Form.Label className={style.resourceText}>Resource:</Form.Label>
-              <ResourceList resourceList={query.resources} titleName={query.setNewResource?.name} />
+              <ResourceList resourceList={query.resources} resourceName={query.setNewResource?.name} />
             </Form.Group>
             <Form.Label className={style.row} />
             <QueryEditor tables={query.setSelectResourceTable} changeCode={changeCode} codeValue={query.setNewCode} />
             <Form.Label className={style.row} />
             <Form.Label className={style.row} />
-            {
-              query.setNewConfirm ? (
-                <Form.Check
-                  type="checkbox"
-                  id="checkbox"
-                  label="Show a confirmation modal before running"
-                  className={style.checkBox}
-                  onClick={changeConfirm}
-                  defaultChecked
-                />
+            <div className={style.checkBoxWrapper}>
+              <div
+                onClick={changeConfirm}
+                onKeyDown={changeConfirm}
+                role="button"
+                tabIndex={0}
+                className={style.Confirm}
+              >
+                {
+              !query.setNewConfirm ? (
+                <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+                  <path d="M5 2c-1.654 0-3 1.346-3 3v14c0 1.654 1.346 3
+                     3 3h14c1.654 0 3-1.346 3-3v-14c0-1.654-1.346-3-3-3h-14zm19
+                     3v14c0 2.761-2.238 5-5 5h-14c-2.762 0-5-2.239-5-5v-14c0-2.761
+                      2.238-5 5-5h14c2.762 0 5 2.239 5 5z"
+                  />
+                </svg>
```

Comment:
don't use raw svg here, it's better to investigate how to fix the problem while still using Form.Check

Reply:
ok